### ISSUE TITLE
android 4.3 status bar is not hidden

### ISSUE THREAD

**Issue by liesislukas** [NishanthShankar/react-native-android-statusbar#5]
Hi, done everything by your readme. No errors. on avd status bar is hidden, on device Galaxy s3 android v4.3 status bar is white-gray color, no icons, but it's not hidden either, it just has no icons + not black background.

**Comment by NishanthShankar** [comment]
Yeah this problem exists in 4.3. I am kind of held up in an other project now. I will solve this as soon as I find the time to. Sorry
